packages/apps/spaces/ui/form/RichTextEditor/menu/TextFormatButtons.tsx

```diff
@@ -1,10 +1,10 @@
 import { useActive, useCommands } from '@remirror/react';
 import { Flex } from '@chakra-ui/react';
-import Bold from '@ui/media/icons/Bold';
-import Italic from '@ui/media/icons/Italic';
-import Strikethrough from '@ui/media/icons/Strikethrough';
+import { Bold01 } from '@ui/media/icons/Bold01';
+import { Italic01 } from '@ui/media/icons/Italic01';
```

Comment:
Could we strip icon names from unnecessary numbers, eg. Bold instead of BoldO1?

Reply:
There is in the library Bold02 which might be needed at some point. The same goes for Heading02 and so on. Stripping the `0` it's something that can become problematic so I would'nt botther.